Sources/SmithyIdentity/BearerTokenIdentityResolvers/StaticBearerTokenIdentityResolver.swift

```diff
@@ -5,11 +5,17 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+@_spi(ClientConfigDefaultIdentityResolver) import protocol SmithyIdentityAPI.ClientConfigDefaultIdentityResolver
```

Comment:
Pretty much the same changes in this `StaticBearerTokenIdentityResolver` as in the `StaticAWSCredentialIdentityResolver` just above.